### PR TITLE
chore: remove thumbnail creation via API

### PR DESCRIPTION
**Pull request by czosel**
This removes support for "manually" uploading thumbnails via the API, because there is no use-case for this (thumbnails should always be created server-side).
Also move the file `create` function from the view to the serializer.

### CODE REVIEW

File: alexandria/core/serializers.py [hunk 1]
```diff
@@ -199,27 +199,20 @@ def get_webdav_url(self, instance):
...
-
-        if (
-            variant := validated_data.get("variant")
-        ) == models.File.Variant.ORIGINAL and validated_data.get("original"):
-            raise ValidationError(
-                f'"original" must not be set for variant "{variant}".'
-            )
 
         mime_type = validated_data["content"].content_type
         if mime_type == "application/octet-stream":
```
Comment (fugal-dy) [inline alexandria/core/serializers.py:218]: the line below should be removed again `validated_data["mime_type"] = validated_data["content"].content_type`

Comment (anehx) [reply alexandria/core/serializers.py:221]: https://github.com/projectcaluma/alexandria/pull/499

Comment (czosel) [reply alexandria/core/serializers.py:221]: This is done in another PR: #499

**Review by anehx — APPROVED** [review]
Nice cleanup! :100: 

Did you test it with the frontend?

File: alexandria/core/views.py [hunk 2]
```diff
@@ -244,43 +244,6 @@ def multi(self, request, **kwargs):
 
             return response
 
-    def create(self, request, *args, **kwargs):
-        # invoke dgap permission check
-        self._check_permissions(request)
```
Comment (Yelinz) [inline alexandria/core/views.py:249]: Should not be removed afaik

**Review by Yelinz — CHANGES_REQUESTED** [review]

**Comment by czosel** [comment]
@Yelinz I noticed that as well, fix is here: https://github.com/adfinis/django-generic-api-permissions/releases/tag/v0.4.4

**Comment by czosel** [comment]
bumped dgap, manual testing with frontend works now

**Review by Yelinz — APPROVED** [review]

**Pull request merged**